scripts/prepare-ci-environment.sh

```diff
@@ -69,3 +69,5 @@ append_intellij_jvm_options() {
 
   echo $cmd
 }
+
+export JDK_JARS="$(printf "%s\n" 'sa-jdi.jar' 'tools.jar')"
```

Comment:
Since `JDK_JARS` in only used in `scripts/run-tests-ci.sh`, you can probably move it there and do not have to export it.

Reply:
You're right, but I also added a reference to that environment var it in `3rdparty/intellij/BUILD` (I moved the usage to the top of the file to make it more clear) since it was getting duplicated there (the `_jdk_jars` variable, now named `_jdk_jar_paths`).